cpp/src/io/orc/reader_impl.cu

```diff
@@ -317,9 +317,12 @@ rmm::device_buffer reader::impl::decompress_stripe_data(
     num_uncompressed_blocks += compinfo[i].num_uncompressed_blocks;
     total_decomp_size += compinfo[i].max_uncompressed_size;
   }
-  CUDF_EXPECTS(total_decomp_size > 0, "No decompressible data found");
+  CUDF_EXPECTS(
+    not((num_compressed_blocks + num_compressed_blocks > 0) and (total_decomp_size == 0)),
```

Comment:
The cleverness of this bit is eluding me:
```c++
(num_compressed_blocks + num_compressed_blocks > 0) and ...
```
Surely `num_compressed_blocks > 0` suffices, no?

Reply:
Good catch. I also didn't realize that :D

Reply:
the names are too similar at first glance :)
Sorry for the unintentional puzzle :D